WORKSPACE

```diff
@@ -129,3 +129,12 @@ rbe_autoconfig(
     # a specific Linux kernel that comes with "libx11" in order to run headless browser tests.
     repository = "google/rbe-ubuntu16-04-webtest",
 )
+
+# Load pinned rules_webtesting browser versions for tests.
+#
+# todo(wagnermaciel): deduplicate browsers - this will load another version of chromium in the
+# repository. We probably want to use the chromium version loaded here (from dev-infra) as that
+# one has RBE improvements.
```

Comment:
when this goes in, can you file an issue to track this?

Reply:
I'm waiting for this too! It has the potential of speeding up local dev too. Right now browser archives are extracted frequently due to bad caching (based on what I see)

Reply:
Just created an issue to make sure this gets done :+1:

Issue #19543